logging/logging_alerts/log-storage-alerts.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="log-storage-alerts"]
+include::_attributes/common-attributes.adoc[]
+= Log storage alerts
+:context: log-storage-alerts
+
+toc::[]
+
+In logging 5.7 and later versions, you can configure your LokiStack deployment to produce log-based alerts.
+
+Application, audit, and infrastructure alerts are sent to the Cluster Monitoring Operator (CMO) Alertmanager in the `openshift-monitoring` namespace by default unless you have disabled the local `Alertmanager` instance.
+
+Application alerts are not sent to the CMO Alertmanager in the `openshift-user-workload-monitoring` namespace by default unless you have enabled a separate `Alertmanager` instance.
```

Comment:
```suggestion
Application alerts are not sent to the Alertmanager used to monitor user-defined projects under the `openshift-user-workload-monitoring` namespace by default unless you have enabled a separate `Alertmanager` instance.
```

Reply:
> unless you have enabled a separate `Alertmanager` instance.

I'm still confused about this part, here are we referring to the dedicated `Alertmanager` Admins can enable for the namespace `openshift-user-workload-monitoring` or are we talking about an external instance of `Alertmanager`? Because if it's the first then I think we should rephrase things differently:
```
Application alerts will be sent to the Alertmanager used to monitor user-defined projects under the `openshift-user-workload-monitoring` namespace if enabled.
```